node/src/apps.rs

```diff
@@ -92,3 +92,46 @@ const NODE_HELP_TEXT: &str = indoc!(
         2. Navigate to the router's port forwarding section, also frequently called virtual server.
         3. Create the port forwarding entries in the router."
 );
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn constants_have_correct_values() {
+        assert_eq!(
```

Comment:
Same as the previous one; here is the question of testing help messages. But maybe that really is a good idea - given it's quite easy to leave an unintended char in text when not paying enough attention.